{
  "comments": [
    {
      "unresolved": true,
      "key": {
        "uuid": "a25aed3f_ea349411",
        "filename": "src/cmd/api/goapi.go",
        "patchSetId": 1
      },
      "lineNbr": 202,
      "author": {
        "id": 6005
      },
      "writtenOn": "2022-09-19T20:20:55Z",
      "side": 0,
      "message": "fail was used to exit with exit code 1 when compareAPI failed. Maybe it should be replaced entirely by the new \u0027exitCode\u0027 that is used in the same way.",
      "revId": "b35b4a31e167a170a5ee481ed525272bc44dcc33",
      "serverId": "62eb7196-b449-3ce5-99f1-c037f21e1705"
    },
    {
      "unresolved": true,
      "key": {
        "uuid": "84fcf2b8_6a685524",
        "filename": "src/cmd/api/goapi.go",
        "patchSetId": 1
      },
      "lineNbr": 206,
      "author": {
        "id": 6005
      },
      "writtenOn": "2022-09-19T20:20:55Z",
      "side": 0,
      "message": "Move os.Exit(exitCode) here, or resolve its poor intersection with the 2 defers in main in some other way.",
      "revId": "b35b4a31e167a170a5ee481ed525272bc44dcc33",
      "serverId": "62eb7196-b449-3ce5-99f1-c037f21e1705"
    }
  ]
}